client/components/mma/accountoverview/ContributionUpdateAmountForm.tsx

```diff
@@ -34,127 +39,8 @@ interface ContributionUpdateAmountFormProps {
 	onUpdateConfirmed: (updatedAmount: number) => void;
 }
 
-export type ContributionInterval = 'month' | 'year';
-
-interface ContributionAmountOptions {
-	amounts: number[];
-	otherDefaultAmount: number;
-	minAmount: number;
-	maxAmount: number;
-}
-
-type ContributionAmountsLookup = Record<
-	string,
-	{
-		month: ContributionAmountOptions;
-		year: ContributionAmountOptions;
-	}
->;
-
 class UpdateAmountLoader extends AsyncLoader<string> {}
 
-// TODO: make this dynamic (i.e. looks up api/config file agreed/shared by contributions team)
```

Comment:
I don't really know what this comment was suggesting so I think it's safe to remove it?

Reply:
I think this is related to this idea to consolidate thresholds for products into one place: https://trello.com/c/2n9CYEEA/3800-spike-consolidate-thresholds-for-product-holdings-in-one-place currently these are separately hardcoded in 5 or 6 (or more) places

Reply:
Oh I see! Perhaps these could also be incorporated in a single place along with the thresholds.

Reply:
Added as a comment to that Trello ticket 👍